tensorflow/python/keras/callbacks.py

```diff
@@ -1325,14 +1327,20 @@ def _save_model(self, epoch, logs):
                         'ModelCheckpoint. Filepath used is an existing '
                         'directory: {}'.format(filepath))
 
-  def _get_file_path(self, epoch, logs):
+  def _get_file_path(self, epoch, batch, logs):
     """Returns the file path for checkpoint."""
     # pylint: disable=protected-access
     try:
       # `filepath` may contain placeholders such as `{epoch:02d}` and
-      # `{mape:.2f}`. A mismatch between logged metrics and the path's
+      # `{batch:02d}`. A mismatch between logged metrics and the path's
       # placeholders can cause formatting to fail.
-      file_path = self.filepath.format(epoch=epoch + 1, **logs)
+      if not batch:
+        file_path = self.filepath.format(epoch=epoch + 1, **logs)
+      else:
+        file_path = self.filepath.format(
+          epoch=epoch + 1,
```

Comment:
Can this fit in one line?

Reply:
No, will give pylint error. But I have changed it to resolve error.